apps/android/app/src/main/java/chat/simplex/app/model/ChatModel.kt

```diff
@@ -23,6 +19,13 @@ class ChatModel(val controller: ChatController, val alertManager: SimplexApp.Ale
   // set when app is opened via contact or invitation URI
   var appOpenUrl = mutableStateOf<Uri?>(null)
 
+  fun updateUserProfile(profile: Profile) {
+    val user = currentUser.value
+    if (user != null) {
+      currentUser = mutableStateOf(user.copy(profile = profile))
```

Comment:
```suggestion
      currentUser.value = user.copy(profile = profile)
```

I think it should work like this, but not sure